control/vehicle_cmd_gate/package.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>vehicle_cmd_gate</name>
+  <version>0.1.0</version>
+  <description>The vehicle_cmd_gate package</description>
+  <maintainer email="h_ohta@ertl.jp">h_ohta</maintainer>
+  <license>Apache 2</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <depend>autoware_control_msgs</depend>
+  <depend>autoware_vehicle_msgs</depend>
+  <depend>rclcpp</depend>
+
```

Comment:
This is missing: 
```
<export>
    <build_type>ament_cmake</build_type>
 </export>
```

which tells `colcon` the type of package. Otherwise, it defaults to `catkin` which will look for libraries (e.g. FastRTPS) in a different way.

Reply:
:+1: Done